app/src/main/java/io/left/reflect/MainActivity.java

```diff
@@ -97,12 +86,12 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         // Send a ping when the floating send button is tapped.
-        View buttonSend = findViewById(R.id.button_send);
-        buttonSend.setOnClickListener(this::sendPing);
+        FloatingActionButton btnSend = findViewById(R.id.button_send);
```

Comment:
In general, it's better to use full words than short forms. e.g. `buttonSend` is more clear than `btnSend`, `listViewLogs` is more clear than `lvLogs`.

Reply:
I tried to use the [android naming convention](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md) but I will change to the full form for easier to get its meaning